model/riscv_vmem_sv48.sail

```diff
@@ -188,13 +188,9 @@ function translate48(asid, ptb, vAddr, ac, priv, mxr, do_sum, level, ext_ptw) =
           TR_Address(pAddr, ext_ptw)
         },
         Some(pbits, ext) =>
-          if not(plat_enable_dirty_update())
-          then {
-            /* pte needs dirty/accessed update but that is not enabled */
-            TR_Failure(PTW_PTE_Update(), ext_ptw)
-          } else {
+          if should_update_PTE_bits() then {
```

Comment:
Is there any reason to swap branch order here instead of using `not(should_update_PTE_bits())`? Not objecting to this just wondering why.

Reply:
I think it's slightly nicer to avoid the not

Reply:
I changed it back, figuring that without a clear consensus, I'd minimize the diff.